internal/controller/job_receiver.go

```diff
@@ -30,7 +31,7 @@ func NewJobReceiver(cm *ConnectionManager, r *mux.Router, kw *kafka.Writer) *Job
 }
 
 func (jr *JobReceiver) Routes() {
-	jr.router.HandleFunc("/job", jr.handleJob())
+	jr.router.HandleFunc("/wss/receptor-controller/job", jr.handleJob())
```

Comment:
At this point, the /job endpoint doesn't need to be public or related to the websocket connection.  I think we could leave the job endpoint as /job or /receptor-controller/job.

Reply:
:+1: